### PR TITLE
Added user signature to textbox by default

### LINKED ISSUE #4: User signature to be added to tickets automatically

### PR DESCRIPTION
**Pull request by joveice**
Adds the user signature (if any) to the textarea in the ticket with two newlines above.
I would argue that this is a better approach than assigning it to the end of the text after submitted, this way you can remove / edit it without anything special features, a well used system for helpdesks.

closes #4

### CODE REVIEW

**Comment by joveice** [comment]
@BadChoice Something like if the text matches the signature only: ignore it?

**Comment by BadChoice** [comment]
Yeah I think is the easiest solution

**Comment by joveice** [comment]
Okey, will look at it when I'm home from work

**Comment by BadChoice** [comment]
Great! thank you :D

**Comment by joveice** [comment]
@BadChoice There, this should do it, feel free to test for yourself.

While doing this I noticed that it returns you back to all tickets after update, maybe it should return back to the ticket? Makes more sense to me that after I update something I get returned back to it.

File: app/Ticket.php [hunk 1]
```diff
@@ -155,8 +155,8 @@ public function addComment($user, $body, $newStatus = null)
         }
         $previousStatus = $this->updateStatusFromComment($user, $newStatus);
         $this->associateUserIfNecessary($user);
-        if (! $body) {
-            return;
+        if (! $body or $body === $user->settings->tickets_signature) {
```
Comment (BadChoice) [inline app/Ticket.php]: I think the &#13 chars will be in the $body so the === will fail,

is that correct? or does it really work this way?

Comment (joveice) [reply app/Ticket.php]: Seems to ignore it, feel free to test yourself, I only did some quick checks.

**Comment by BadChoice** [comment]
Thaaaanks!

**Pull request merged**